deploy/dashboards/metrics/ibm-websphere-liberty-grafana-dashboard.json

```diff
@@ -51,1901 +15,1991 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
-  "id": null,
-  "iteration": 1558445746439,
+  "id": 2,
+  "iteration": 1569269817098,
   "links": [],
   "panels": [
     {
-      "collapsed": true,
+      "collapsed": false,
```

Comment:
dashboard should be exported with all rows collapsed

Reply:
I made one commit to collapse the dashboard. As well as another one to reapply the __inputs for the Prometheus datasource and version __require data that were lost from exporting.